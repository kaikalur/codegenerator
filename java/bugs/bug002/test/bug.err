Encountered " "}" "} "" at line 1, column 3.
Was expecting:
    "(" ...
    
