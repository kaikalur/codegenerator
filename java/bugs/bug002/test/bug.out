Call: EnumerationItem
  Consumed token: <"{" at line 1 column 1>
  Call: NamedNumber(LOOKING AHEAD...)
    Visited token: <<IDENTIFIANT>: "c" at line 1 column 2>; Expected token: <<IDENTIFIANT>>
  Return: NamedNumber(LOOKAHEAD SUCCEEDED)
  Call: NamedNumber
    Call: identifier
      Consumed token: <<IDENTIFIANT>: "c" at line 1 column 2>
    Return: identifier
  Return: NamedNumber
Return: EnumerationItem
